tests/baselines/reference/genericUnknownTypeParamWithKeyofIndexedAccess.errors.txt

```diff
@@ -0,0 +1,26 @@
+tests/cases/compiler/genericUnknownTypeParamWithKeyofIndexedAccess.ts(2,12): error TS2531: Object is possibly 'null'.
+tests/cases/compiler/genericUnknownTypeParamWithKeyofIndexedAccess.ts(5,12): error TS2345: Argument of type 'string' is not assignable to parameter of type 'never'.
+tests/cases/compiler/genericUnknownTypeParamWithKeyofIndexedAccess.ts(8,12): error TS2531: Object is possibly 'null'.
+tests/cases/compiler/genericUnknownTypeParamWithKeyofIndexedAccess.ts(11,13): error TS2345: Argument of type 'string' is not assignable to parameter of type 'never'.
+
+
+==== tests/cases/compiler/genericUnknownTypeParamWithKeyofIndexedAccess.ts (4 errors) ====
+    function test<T, K extends keyof T>(t: T, k: K) {
+        return t[k];
+               ~
+!!! error TS2531: Object is possibly 'null'.
+    }
+    
+    test(null, 'foo')

```

Comment:
I've included those calls as there was a temporary regression related to this, you can check out [here](https://github.com/microsoft/TypeScript/pull/49696#discussion_r907811832). I've figured out that this might be worth it to add here as part of the regression tests